backend/src/main/java/ai/verta/modeldb/project/FutureProjectDAO.java

```diff
@@ -63,9 +63,11 @@ public FutureProjectDAO(Executor executor, FutureJdbi jdbi, UAC uac) {
 
   public InternalFuture<List<KeyValue>> getAttributes(GetAttributes request) {
     final var projectId = request.getId();
+    final var keys = request.getAttributeKeysList();
+    final var getAll = request.getGetAll();
 
     return checkProjectPermission(projectId, ModelDBActionEnum.ModelDBServiceActions.READ)
-        .thenCompose(unused -> attributeHandler.getKeyValues(projectId), executor);
+        .thenCompose(unused -> attributeHandler.getKeyValues(projectId, keys, getAll), executor);
```

Comment:
while getting attributes via `getAttributes ` forget to considered `getAttributeKeysList`, and `getAll` flag which is added now.